src/main/java/com/google/sps/utility/KeyProvider.java

```diff
@@ -29,7 +29,11 @@
 
   /** Constructor to create instance with file linked to default path. */
   public KeyProvider() {
-    this.file = new File("src/main/resources/KEYS.json");
+    if (System.getProperty("user.dir").contains("target")) {
```

Comment:
Please add a comment explaining why this is being necessary

Reply:
I have added a comment, let me know if that makes sense!

Reply:
Tks!
RESOLVED